enhancements/network/user-defined-network-segmentation.md

```diff
@@ -521,13 +521,22 @@ type UserDefinedNetworkSpec struct {
     // Supported by Topology `Layer2` and `Localnet`.
     // +optional
     IPAMLifecycle NetworkIPAMLifecycle `json:"ipamLifecycle,omitempty"`
+    
+    // Control how much of the IP configuration will be managed by OVN.
+    // When `FullyManaged`, OVN will apply IP configuration to the SDN infrastructure and it will also assign IPs from the selected subnet to the individual pods.
+    // When `Disabled`, OVN will only provide layer 2 communication, letting users configure IP addresses for the pods. `Disabled` is only available for `Layer2`
```

Comment:
do we want to highlight which features the users would effectively opt-out of when they disable IPAM ? If I'm a clueless user, I would like to have that info shoved in my face before making what I think is an harmless choice. 

- IP spoofing protection
- namespace / pod selectors in multi-net policies

are the ones that immediately come to mind.

Reply:
I like that idea, I will add it